pkg/osquery/extension.go

```diff
@@ -215,94 +206,6 @@ func (e *Extension) getHostIdentifier() (string, error) {
 	return IdentifierFromDB(e.knapsack.ConfigStore(), e.registrationId)
 }
 
-// SetupLauncherKeys configures the various keys used for communication.
-//
-// There are 3 keys:
-// 1. The RSA key. This is stored in the launcher DB, and was the first key used by krypto. We are deprecating it.
-// 2. The hardware keys -- these are in the secure enclave (TPM or Apple's thing) These are used to identify the device
-// 3. The launcher install key -- this is an ECC key that is sometimes used in conjunction with (2)
```

Comment:
I think this comment is old -- we don't generate the ECC keys in this function. Thus it's removal is safe